pkg/services/ngalert/schedule/schedule_unit_test.go

```diff
@@ -59,7 +59,7 @@ func TestProcessTicks(t *testing.T) {
 	mockedClock := clock.NewMock()
 
 	notifier := &AlertsSenderMock{}
-	notifier.EXPECT().Send(mock.Anything, mock.Anything).Return()
+	notifier.EXPECT().Send(context.Background(), mock.Anything, mock.Anything).Return()
```

Comment:
The only change in this file is adding a `context.Context` to `.Send()` calls.
The same goes for `router.go` and `ruter_test.go`